build.gradle.kts

```diff
@@ -0,0 +1,75 @@
+version = "0.1.1"
+apply(plugin = "java-library")
+
+plugins {
+    id("org.jetbrains.kotlin.jvm") version Versions.kotlin_plugin
+    id("io.gitlab.arturbosch.detekt").version(Versions.detekt)
+
+    `java-library`
+}
+
+repositories {
+    mavenCentral()
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
+    kotlinOptions {
+        jvmTarget = "11"
+    }
+}
+
+dependencies {
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
+
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+
+    implementation("com.google.guava:guava:${Versions.guava}")
+    implementation("org.junit.jupiter:junit-jupiter:${Versions.junit_jupiter}")
+    implementation("com.navelplace:jsemver:${Versions.jsemver}")
```

Comment:
we shouldn't be neeeding this dependency (it's a [version range parser](https://github.com/BrianDeacon/jsemver) that is required only because i used `stripWhitespace()` function in tests). 

Please add following util function to test directory:
```
fun String.stripWhitespace(): String {
    return this
            .replace(" ", "")
            .replace("\n", "")
            .replace("\t", "")
}
```
and remove this dependency

Reply:
done